### PR TITLE
Legal pages

### PR DESCRIPTION
**Pull request by nilshindermann**
Adds new pages:

- `/imprint`
- `/privacy`

Other changes:

- Update links in the footer.
- Navigation bar is sticky now.
- Email address is obfuscated so that bots cannot read it.

### CODE REVIEW

File: components/ContactEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+interface ContactEmailProps {
+    email: string;
+}
+
+export default function ContactEmail({
+    email,
+}: ContactEmailProps): ReactElement {
+    return (
+        <Obfuscate
```
Comment (llettieri) [inline components/ContactEmail.tsx:14]: What does this `Obfuscate` do?

**Review by llettieri — DISMISSED** [review]

File: components/ContactEmail.tsx (hunk 1 above)
Comment (nilshindermann) [reply components/ContactEmail.tsx:14]: Reverse the text until a user interacts with it (eg. hover over it)

More details here: https://github.com/South-Paw/react-obfuscate-ts

File: app/(default)/(legal)/legal.css
Comment (nilshindermann) [inline app/(default)/(legal)/legal.css:1]: This is problematic, need to figure this out

Comment (nilshindermann) [reply app/(default)/(legal)/legal.css:1]: This has been fixed in b2bde79d5e37aa2ab06da25ea73ff20bf91da098

**Review by llettieri — APPROVED** [review]

**Pull request merged**